### PR TITLE
git-guilt: add page

### PR DESCRIPTION
**Pull request by Sadeeed**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

#5137

### CODE REVIEW

File: pages/common/git-guilt.md
Comment (reinhart1010) [inline pages/common/git-guilt.md:13]:
```suggestion
`git guilt {{first_revision}} {{last_revision}}`
```

**Review by reinhart1010 — CHANGES_REQUESTED** [review]
Thanks for contributing! Here are some of my notes to review.

**Review by reinhart1010 — APPROVED** [review]
Thanks! We need to wait naother maintainer to approve this before finally merge this PR.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**
